java/application-insights/README.md

```diff
@@ -2,33 +2,31 @@
 
 ## Binding
 
-The buildpack installs the Azure Application Insights Agent, and configures it for usage based on a Service Binding.  This binding consists of a `kind` indicating what type of service it is, and an `InstrumentationKey` with the Application Insight Instrumentation Key.
+The buildpack installs the Azure Application Insights Agent, and configures it for usage based on a Service Binding.  This binding consists of a `type` indicating what type of service it is, and an `InstrumentationKey` with the Application Insight Instrumentation Key.
 
 ```plain
 binding
-├── metadata
-│   ├── kind
-│   └── provider
-└── secret
-    └── InstrumentationKey
+├── type
+├── provider
```

Comment:
I know how you hate it; do you want to remove `provider` (and the matching file)?

Reply:
It would be my pleasure